CI failures in MSRV jobs failing to pick up ahash version

https://github.com/chronotope/chrono/actions/runs/6678120652/job/18148831947?pr=1343

```
Run cargo test --lib \
    Updating crates.io index
error: failed to select a version for the requirement `ahash = "^0.7.0"`
candidate versions found which didn't match: 0.4.8, 0.4.1, 0.4.0, ...
location searched: crates.io index
required by package `hashbrown v0.12.0`
    ... which satisfies dependency `hashbrown = "^0.12"` of package `rkyv v0.7.41`
    ... which satisfies dependency `rkyv = "^0.7.41"` of package `chrono v0.4.31 (/home/runner/work/chrono/chrono)`
Error: Process completed with exit code 101.
```

It's weird because ahash 0.7.7 was released 5 days ago but apparently isn't getting picked up here?

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -19,7 +19,7 @@ jobs:
       - run: cargo test --all-features --color=always -- --color=always
 
   # later this may be able to be included with the below
-  # kept separate for now as the following don't compile on 1.57
+  # kept separate for now as the following don't compile on 1.60
   # * arbitrary (requires 1.63 as of v1.3.0)
   rust_msrv:
     strategy:
@@ -30,7 +30,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@master
         with:
-          toolchain: 1.57
+          toolchain: "1.61.0"
       - uses: Swatinem/rust-cache@v2
       # run --lib and --doc to avoid the long running integration tests
       # which are run elsewhere

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,7 +11,7 @@ readme = "README.md"
 license = "MIT OR Apache-2.0"
 exclude = ["/ci/*"]
 edition = "2021"
-rust-version = "1.57.0"
+rust-version = "1.61.0"
 
 [lib]
 name = "chrono"

--- a/README.md
+++ b/README.md
@@ -67,7 +67,7 @@ Optional features:
 
 ## Rust version requirements
 
-The Minimum Supported Rust Version (MSRV) is currently **Rust 1.57.0**.
+The Minimum Supported Rust Version (MSRV) is currently **Rust 1.61.0**.
 
 The MSRV is explicitly tested in CI. It may be bumped in minor releases, but this is not done
 lightly.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -359,7 +359,7 @@
 //!
 //! ## Rust version requirements
 //!
-//! The Minimum Supported Rust Version (MSRV) is currently **Rust 1.57.0**.
+//! The Minimum Supported Rust Version (MSRV) is currently **Rust 1.61.0**.
 //!
 //! The MSRV is explicitly tested in CI. It may be bumped in minor releases, but this is not done
 //! lightly.

